2025-10-18 22:17:35,938 WARNING: Favicon not found: /favicon.ico [in C:\Users\Sub\Documents\GitHub\Document_Compliance_System\app.py:427]
2025-10-18 22:17:49,026 WARNING: Favicon not found: /favicon.ico [in C:\Users\Sub\Documents\GitHub\Document_Compliance_System\app.py:427]
2025-10-18 22:17:50,734 WARNING: Favicon not found: /favicon.ico [in C:\Users\Sub\Documents\GitHub\Document_Compliance_System\app.py:427]
2025-10-18 22:17:52,338 WARNING: Favicon not found: /favicon.ico [in C:\Users\Sub\Documents\GitHub\Document_Compliance_System\app.py:427]
2025-10-18 22:17:55,499 WARNING: Favicon not found: /favicon.ico [in C:\Users\Sub\Documents\GitHub\Document_Compliance_System\app.py:427]
2025-10-18 22:18:06,115 ERROR: User: 1 - An error occurred during file upload: 
**********************************************************************
  Resource [93mpunkt_tab[0m not found.
  Please use the NLTK Downloader to obtain the resource:

  [31m>>> import nltk
  >>> nltk.download('punkt_tab')
  [0m
  For more information see: https://www.nltk.org/data.html

  Attempted to load [93mtokenizers/punkt_tab/english/[0m

  Searched in:
    - 'C:\\Users\\Sub/nltk_data'
    - 'C:\\Users\\Sub\\AppData\\Local\\Programs\\Python\\Python39\\nltk_data'
    - 'C:\\Users\\Sub\\AppData\\Local\\Programs\\Python\\Python39\\share\\nltk_data'
    - 'C:\\Users\\Sub\\AppData\\Local\\Programs\\Python\\Python39\\lib\\nltk_data'
    - 'C:\\Users\\Sub\\AppData\\Roaming\\nltk_data'
    - 'C:\\nltk_data'
    - 'D:\\nltk_data'
    - 'E:\\nltk_data'
**********************************************************************
 - Template file: N/A, Student file: Development of a Finance management System project proposal.docx, Request: POST /upload [in C:\Users\Sub\Documents\GitHub\Document_Compliance_System\app.py:249]
2025-10-18 22:18:06,209 WARNING: Favicon not found: /favicon.ico [in C:\Users\Sub\Documents\GitHub\Document_Compliance_System\app.py:427]
2025-10-18 22:27:03,497 INFO: Application startup [in C:\Users\Sub\Documents\GitHub\Document_Compliance_System\app.py:42]
2025-10-18 22:27:22,561 WARNING: Favicon not found: /favicon.ico [in C:\Users\Sub\Documents\GitHub\Document_Compliance_System\app.py:427]
2025-10-18 22:27:29,631 WARNING: Favicon not found: /favicon.ico [in C:\Users\Sub\Documents\GitHub\Document_Compliance_System\app.py:427]
2025-10-18 22:27:31,953 WARNING: Favicon not found: /favicon.ico [in C:\Users\Sub\Documents\GitHub\Document_Compliance_System\app.py:427]
2025-10-18 22:27:51,053 INFO: Saving template path: uploads\Development_of_a_Finance_management_System_project_proposal.docx [in C:\Users\Sub\Documents\GitHub\Document_Compliance_System\app.py:127]
2025-10-18 22:27:51,512 ERROR: User: 1 - An error occurred during file upload: 
**********************************************************************
  Resource [93mpunkt_tab[0m not found.
  Please use the NLTK Downloader to obtain the resource:

  [31m>>> import nltk
  >>> nltk.download('punkt_tab')
  [0m
  For more information see: https://www.nltk.org/data.html

  Attempted to load [93mtokenizers/punkt_tab/english/[0m

  Searched in:
    - 'C:\\Users\\Sub/nltk_data'
    - 'C:\\Users\\Sub\\AppData\\Local\\Programs\\Python\\Python39\\nltk_data'
    - 'C:\\Users\\Sub\\AppData\\Local\\Programs\\Python\\Python39\\share\\nltk_data'
    - 'C:\\Users\\Sub\\AppData\\Local\\Programs\\Python\\Python39\\lib\\nltk_data'
    - 'C:\\Users\\Sub\\AppData\\Roaming\\nltk_data'
    - 'C:\\nltk_data'
    - 'D:\\nltk_data'
    - 'E:\\nltk_data'
**********************************************************************
 - Template file: Development of a Finance management System project proposal.docx, Student file: Development of a Finance management System project proposal.docx, Request: POST /upload [in C:\Users\Sub\Documents\GitHub\Document_Compliance_System\app.py:249]
2025-10-18 22:27:51,588 WARNING: Favicon not found: /favicon.ico [in C:\Users\Sub\Documents\GitHub\Document_Compliance_System\app.py:427]
2025-10-18 22:42:38,075 INFO: Application startup [in C:\Users\Sub\Documents\GitHub\Document_Compliance_System\app.py:42]
2025-10-18 22:42:46,047 WARNING: Favicon not found: /favicon.ico [in C:\Users\Sub\Documents\GitHub\Document_Compliance_System\app.py:427]
2025-10-18 22:42:57,644 WARNING: Favicon not found: /favicon.ico [in C:\Users\Sub\Documents\GitHub\Document_Compliance_System\app.py:427]
2025-10-18 22:42:59,281 WARNING: Favicon not found: /favicon.ico [in C:\Users\Sub\Documents\GitHub\Document_Compliance_System\app.py:427]
2025-10-18 22:43:16,568 INFO: Saving template path: uploads\Development_of_a_Finance_management_System_project_proposal.docx [in C:\Users\Sub\Documents\GitHub\Document_Compliance_System\app.py:127]
2025-10-18 22:43:17,105 ERROR: User: 1 - An error occurred during file upload: 
**********************************************************************
  Resource [93mpunkt_tab[0m not found.
  Please use the NLTK Downloader to obtain the resource:

  [31m>>> import nltk
  >>> nltk.download('punkt_tab')
  [0m
  For more information see: https://www.nltk.org/data.html

  Attempted to load [93mtokenizers/punkt_tab/english/[0m

  Searched in:
    - 'C:\\Users\\Sub/nltk_data'
    - 'C:\\Users\\Sub\\AppData\\Local\\Programs\\Python\\Python39\\nltk_data'
    - 'C:\\Users\\Sub\\AppData\\Local\\Programs\\Python\\Python39\\share\\nltk_data'
    - 'C:\\Users\\Sub\\AppData\\Local\\Programs\\Python\\Python39\\lib\\nltk_data'
    - 'C:\\Users\\Sub\\AppData\\Roaming\\nltk_data'
    - 'C:\\nltk_data'
    - 'D:\\nltk_data'
    - 'E:\\nltk_data'
**********************************************************************
 - Template file: Development of a Finance management System project proposal.docx, Student file: Development of a Finance management System project proposal.docx, Request: POST /upload [in C:\Users\Sub\Documents\GitHub\Document_Compliance_System\app.py:249]
2025-10-18 22:43:17,198 WARNING: Favicon not found: /favicon.ico [in C:\Users\Sub\Documents\GitHub\Document_Compliance_System\app.py:427]
